src/Dfe.PlanTech.AzureFunctions/Mappings/RecommendationIntroUpdater.cs

```diff
@@ -28,8 +28,7 @@ public override MappedEntity UpdateEntityConcrete(MappedEntity entity)
     private void RemoveOldAssociatedIntroContents(RecommendationIntroDbEntity existingRecommendationIntro)
     {
         var contentsToRemove = Db.RecommendationIntroContents
-            .Where(content => content.RecommendationIntroId == existingRecommendationIntro.Id)
-            .ToList();
+            .Where(content => content.RecommendationIntroId == existingRecommendationIntro.Id);
 
         Db.RecommendationIntroContents.RemoveRange(contentsToRemove);
```

Comment:
same

Reply:
Keeping consistent and just using the local variable for each one.